docs/execplans/afl-circular-dependency-issue.md

```
@@ -116,10 +116,8 @@ contains an executable `/usr/local/bin/fuzz` binary that AFL++ can run.
 - Observation: the required repository commit gates are presently blocked by
   environment/tooling issues unrelated to this Dockerfile change. Evidence:
   `make lint` fails in the `whitaker` step with
-  `error: unknown start of token: \`` while probing a nightly toolchain, and
-  `make test` fails during the postgres nextest path because
-  `/root/.rustup/toolchains/nightly-2025-11-08-x86_64-unknown-linux-gnu/bin/rustc`
-   is missing. Impact: this change cannot be fully validated in the current
+  ``` error: unknown start of token: \`` while probing a nightly toolchain, and  ```make test`fails during the postgres nextest path because`/root/.rustup/toolchains/nightly-2025-11-08-x86_64-unknown-linux-gnu/bin/rustc\`
```

markdownlint-cli2 (0.22.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)